### DISCUSSION TITLE
Support for "target" attribute in application.conf in custom nav and menu links.

### DISCUSSION THREAD

**Discussion by rberends** [Erudika/scoold#315] [Ideas]
Hey, for the custom links that are available for the menus, it would be great if there would be support to also customize the behaviour of the links through the "target" attribute, in the application.conf.

I already have a fork with the needed changes here, and would pull request it - but I'm having some trouble getting the whitespace/line endings right for base.vm on my Windows machine, which now shows a massive dif.

[Link to fork](https://github.com/rberends/scoold-custom-link-target)

**Comment by albogdano** [comment]
Thank you! I will review the changes and merge those in `master` tomorrow.

**Reply by rberends** [reply]
Sure! Let me know if any changes need to be made before this would be merge-worthy.
